### PR TITLE
Prevent JavaScript SyntaxError if vars are undefined

### PR DESCRIPTION
**Pull request by JasonCook599**
Resolves xibosignage/xibo#2698
Resolves xibosignage/xibo#2699

### CODE REVIEW

**Comment by dasgarner** [comment]
Thanks Jason, I've created two issues to track these changes so we can more easily see what was changes and why (and to pull into the release notes).

https://github.com/xibosignage/xibo/issues/2698
https://github.com/xibosignage/xibo/issues/2699

I'll review the changes shortly! Happy New Year!

File: views/dashboard-status-page.twig [hunk 1]
```diff
@@ -450,7 +450,7 @@
             }
         });
 
-        var displayMediaStatusData = {{ displayMediaStatus|raw }};
+        var displayData {% if displayStatus|raw %}= {{ displayStatus|raw }}{% endif %};
```
Comment (dasgarner) [inline views/dashboard-status-page.twig:453]: Should this actually be as below?

```suggestion
        var displayMediaStatusData {% if displayMediaStatus|raw %}= {{ displayMediaStatus|raw }}{% endif %};
```

**Review by dasgarner — CHANGES_REQUESTED** [review]

**Comment by dasgarner** [comment]
There is one bit that didn't quite make sense to me - the rest looks great!

We're really happy to have your contributions, please let me know if I can help you in anyway.

File: views/dashboard-status-page.twig (hunk 1 above)
Comment (JasonCook599) [reply views/dashboard-status-page.twig:453]: You are correct. I think I just copy-pasted incorrectly.

**Review by dasgarner — APPROVED** [review]

**Pull request merged**